pyproject.toml

```diff
@@ -1,15 +1,58 @@
-[build-system]
-requires = ["setuptools<72.2", "wheel", "numpy>=1.15", "Cython>=0.29"]
+[project]
+name = "pymoo"
+description = "Multi-Objective Optimization in Python"
+authors = [{ name = "Julian Blank", email = "blankjul@outlook.com" }]
+readme = "README.rst"
+license = "Apache-2.0"
+classifiers = [
+    'Intended Audience :: Developers',
+    'Intended Audience :: Science/Research',
+    'Operating System :: OS Independent',
```

Comment:
NB I removed the "License" classifier, as it is now deprecated. setuptools otherwise spits out noisy warnings like this when building: https://github.com/bokeh/bokeh/issues/14434